src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -584,6 +626,11 @@ object FastSync {
   validators: Validators, peerEventBus: ActorRef, etcPeerManager: ActorRef, syncConfig: SyncConfig, scheduler: Scheduler): Props =
     Props(new FastSync(fastSyncStateStorage, appStateStorage, blockchain, validators, peerEventBus, etcPeerManager, syncConfig, scheduler))
 
+  // validation parameters (see: https://github.com/ethereum/go-ethereum/pull/1889)
```

Comment:
Even if these values seem to be defined in the referenced PR, shouldn't be a good idea to put them within application.conf? (hidden from the normal user)

Reply:
done